octavia_f5/common/config.py

```diff
@@ -48,8 +48,8 @@ def setup_logging(conf):
                 help=_('Use token authentication.')),
     cfg.BoolOpt('bigip_verify', default=False,
                 help=_('Verify AS3 endpoint TLS cert.')),
-    cfg.StrOpt('bigip_url',
-               help=_('The URL to the bigip host device with AS3 endpoint')),
+    cfg.StrOpt('bigip_url_a', help=_('The URL to the bigip host device with AS3 endpoint')),
+    cfg.StrOpt('bigip_url_b', help=_('The URL to the bigip host device with AS3 endpoint')),
```

Comment:
Please use a more generic list for the definition of the urls, e.g.: https://github.com/sapcc/networking-f5/blob/master/networking_f5/agent/f5_agent.py#L62